stable/mongodb/README.md

```diff
@@ -92,9 +105,36 @@ Alternatively, a YAML file that specifies the values for the parameters can be p
 ```bash
 $ helm install --name my-release -f values.yaml stable/mongodb
 ```
-
 > **Tip**: You can use the default [values.yaml](values.yaml)
 
+## Replication
+
+You can start the MongoDB chart in replica set mode with the following command:
+
+```bash
+$ helm install --name my-release stable/mongodb --set replication.enabled=true
```

Comment:
This `replication.enabled=true` is incorrect. It should be `replicaSet.enabled=true`